rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,6 +16,10 @@ toc::[]
 [id="rosa-q3-2023_{context}"]
 === Q3 2023
 
+* **ROSA CLI update:** The ROSA CLI (`rosa`) was updated to a new version. For information about what's changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.27[release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc[Getting started with the ROSA CLI].
+
+* **Cluster autoscaling:** You can now enable cluster autoscaling using ROSA clusters. CLuster autoscaling automatically adjusts the size of a cluster so that all pods have a place to run and there are no unneeded nodes. You can enable autoscaling during and after cluster creation using either OpenShift Cluster Manager or the ROSA CLI (`rosa`). For more information, see xref:../rosa_cluster_admin/rosa-cluster-autoscaling.html[Cluster autoscaling].
```

Comment:
Extra caps on the L.

s/CLuster autoscaling automatically/Cluster autoscaling automatically